src/include/migraphx/operation.hpp

```diff
@@ -1097,6 +1173,12 @@ struct operation
             return private_detail_te_default_has_finalize(char(0), private_detail_te_value);
         }
 
+        bool has_offload_copy() const override
+        {
+
```

Comment:
Remove whitespace here

Reply:
This code has been auto-generated. It has extra space because  common logic for the auto-gen was supposed to put namespace identifications e.g. `using ` clause there but in this method, it doesn't need any namespace identification and therefore emtpy. I can put a check for empty `using` clause somehow but would like to keep it for some other PR.